### PR TITLE
Update EasternKingdoms.lua

### PR DESCRIPTION
**Pull request by anon1231823**

### CODE REVIEW

File: Localization/Translations/Zones/EasternKingdoms.lua [hunk 1]
```diff
@@ -327,14 +327,14 @@ local easternKingdomsLocales = {
         ["zhCN"] = "奥特兰克山脉",
     },
     ["Dalaran"] = {
-        ["ptBR"] = true,
+        ["ptBR"] = "Dalaran",
```
Comment (BreakBB) [inline Localization/Translations/Zones/EasternKingdoms.lua:330]: `true` always means "take the key as translation". So if the key and the translation are the same it is better to leave it as `true`.

Comment (anon1231823) [reply Localization/Translations/Zones/EasternKingdoms.lua:330]: That shipped sailed a while ago, every other translation put it in

Comment (BreakBB) [reply Localization/Translations/Zones/EasternKingdoms.lua:330]: No reason to make it worse though.

**Comment by BreakBB** [comment]
Thanks for coming back to this @anon1231823 💪🏻 Great work, thank you! :)

**Pull request merged**